src/SFML/Network/Http.cpp

```diff
@@ -369,7 +369,7 @@ Http::Response Http::sendRequest(const Http::Request& request, Time timeout)
     Response received;
 
     // Connect the socket to the host
-    if (m_connection.connect(m_host, m_port, timeout) == Socket::Done)
+    if (m_connection.connect(m_host.value(), m_port, timeout) == Socket::Done)
```

Comment:
Do we want to check this contains a value to avoid risking a `std::bad_optional_access`? Maybe just `assert` that it has a value?

Reply:
Was there ever a conclusion made about this or is it not possible for `.value()` to be called on a null optional here?